web/themes/custom/cloudy/package-lock.json

```diff
@@ -8,7 +8,6 @@
       "version": "7.0.0",
       "resolved": "https://registry.npmjs.org/@babel/code-frame/-/code-frame-7.0.0.tgz",
       "integrity": "sha512-OfC2uemaknXr87bdLUkWog7nYuliM9Ij5HUcajsVcMCpQrcLmtxRbVFTIqmcSkSeYRBFBRxs2FiUqFJDLdiebA==",
-      "dev": true,
```

Comment:
I'm not sure why the package-lock.json was touched with this change. Seems like it should not have changed unless you ran an npm update. This story should not have required that should it?

Reply:
We talked about this in standup. The likely difference was running npm from your machine instead of within lando. You should always use `lando npm` commands rather than using locally installed npm. Let me know if that was not the case as we might need to update our lando node containers.